db/migrate/20170221195674_tidy_current_roles.rb

```diff
@@ -21,34 +21,10 @@ def up
   def process_existing(original_name, role, permission_names)
     diff = role.permission_diff permission_names
     return if diff.empty?
-    rename_existing role, original_name
+    SeedHelper.rename_existing_role role, original_name
     create_from_seeds original_name, permission_names
   end
 
-  def rename_existing(role, original_name)
```

Comment:
If we want to keep a copy in here, we should also have copies of `rename_with_free` and `generate_name` methods, because the same applies to them. I decided to extract these because copy-pasting 3 methods from different migration seemed ugly.